packages/block-editor/src/components/child-layout-control/index.js

```diff
@@ -104,37 +151,93 @@ export default function ChildLayoutControl( {
 							value={ flexSize }
 						/>
 					) }
-				</>
+				</VStack>
 			) }
 			{ parentLayoutType === 'grid' && (
-				<HStack>
-					<InputControl
-						size={ '__unstable-large' }
-						label={ __( 'Column Span' ) }
-						type="number"
-						onChange={ ( value ) => {
-							onChange( {
-								rowSpan,
-								columnSpan: value,
-							} );
-						} }
-						value={ columnSpan }
-						min={ 1 }
-					/>
-					<InputControl
-						size={ '__unstable-large' }
-						label={ __( 'Row Span' ) }
-						type="number"
-						onChange={ ( value ) => {
-							onChange( {
-								columnSpan,
-								rowSpan: value,
-							} );
-						} }
-						value={ rowSpan }
-						min={ 1 }
-					/>
-				</HStack>
+				<>
+					{ window.__experimentalEnableGridInteractivity && (
+						<HStack
+							as={ ToolsPanelItem }
+							hasValue={ hasStartValue }
+							label={ __( 'Grid placement' ) }
+							onDeselect={ resetGridStarts }
+							isShownByDefault={ false }
+							panelId={ panelId }
+						>
+							<InputControl
+								size={ '__unstable-large' }
+								label={ __( 'Column' ) }
```

Comment:
Oh, just double-checking, should the visual label on these control also use the word `placement`, so that it matches the menu item?

Reply:
It would make things clearer! It's a pretty long string, though I'm not sure how relevant that is given that strings can have any length in translation.

Reply:
I don't think it's necessary. Also if we say _Column placement_ and _Row placement_ the labels truncate as they're too long.

I suspect we'll change the labels (and lots of other things) based on user testing.